zcash_primitives/Cargo.toml

```diff
@@ -42,11 +42,15 @@ subtle = "2.2.3"
 bls12_381 = "0.8"
 ff = "0.13"
 group = { version = "0.13", features = ["wnaf-memuse"] }
-incrementalmerkletree = "0.3"
 jubjub = "0.10"
 nonempty = "0.7"
 orchard = { version = "0.4", default-features = false }
 
+# - Note Commitment Trees
+bridgetree = "0.2"
+either = "1.8"
```

Comment:
I don't recall seeing this import anywhere in this PR. What is it a dependency for?